packages/code-studio/src/settings/SettingsMenu.tsx

```diff
@@ -163,7 +163,7 @@ export class SettingsMenu extends Component<
                   transform="grow-4"
                   className="mr-2"
                 />
-                Column Specific Formatting Rules
+                Format by Column Name & Type
```

Comment:
We use `&amp;` above, but only `&` here. Should probably be consistent?
```suggestion
                Format by Column Name &amp; Type
```